install.sh

```diff
@@ -142,7 +143,12 @@ install_dependencies() {
     apt-key adv --keyserver keyserver.ubuntu.com --recv-keys AE33835F504A1A25
     apt-get update
     apt-get install -y wireguard
-    apt-get install -y libmnl-dev libelf-dev linux-headers-$(uname -r) build-essential pkg-config
+    apt-get install -y libmnl-dev libelf-dev build-essential pkg-config
+    if [[ "$distro" == "raspbian" ]]; then
+      apt-get install -y raspberrypi-kernel-headers
```

Comment:
There are 4 spaces in other places, let's keep it consistent. 
```suggestion
        apt-get install -y raspberrypi-kernel-headers
```